### DISCUSSION TITLE
Alienfx-gui:  “Error Fan control start failure, disabling fan control”. What can I do?

### DISCUSSION THREAD

**Discussion by EstebanRoberto** [T-Troll/alienfx-tools#458] [Q&A]
The fan control does not work, I get “Error Fan control start failure, disabling fan control”. What can I do?
I have a Dell G15 5525 with AMD Ryzen 7

![Captura de pantalla (1)](https://github.com/user-attachments/assets/422e591c-a894-4444-8c06-3aa4a7704adf)

**Comment by T-Troll** [comment]
Exactly as typed - it was an error then trying to start fan control.
Can you please open Administrator CMD and run command (from the tools folder) - `alienfan-cli dump`? Then share result for me.

**Reply by EstebanRoberto** [reply]
I get this message

![image](https://github.com/user-attachments/assets/fc46de62-9e5f-4d7d-b64c-34f3b74662b9)

**Reply by T-Troll** [reply]
"Fan SDK_V1". Did you even read release notes? It printed BIG LETTERS, do not use if you don't sure you know what you're doing!
